src/content/en/updates/2018/12/badging-api.md

```diff
@@ -148,14 +156,22 @@ in this case, the browser will attempt to provide the best representation for
 that device. For example, while the Badging API isn’t supported on Android,
 Android only ever shows a dot instead of a numeric value.
 
+Note: Don’t assume anything about how the user agent wants to display the badge.
+We expect some user agents will take a number like "4000" and rewrite it as "99+".
+If you saturate it yourself (for example to "99") then the "+" won’t appear.
+No matter the actual number, just set `Badge.set(unreadCount)`
+and let the user agent deal with displaying it accordingly.
+
 ## Feedback {: #feedback }
 
 We need your help to ensure that the Badging API works in a way that meets your
 needs and that we’re not missing any key scenarios.
 
 <aside class="key-point">
-  <b>We need your help!</b> - Will the current design (allowing an integer
-  from 1-99, 99+, or a white dot/flag) meet your needs? If it won’t, please
+  <b>We need your help!</b> - Will the current design (allowing any integer
+  that *in Chrome* gets displayed as 1–99 or 99+,
```

Comment:
I don't think we should call out Chrome's behaviour. We don't want developers writing for Chrome, we want them writing for the API we're presenting. The decision about how the integer is displayed isn't up to Chrome, it's up to the underlying OS, so it will be different on Windows, macOS, Chrome OS, etc. That should be irrelevant to developers (as you wrote above). Just set a number and _something_ appropriate will be displayed.

All we should be asking here is "will the current design (allowing either an integer or a flag value) meet your needs".